build.gradle.kts

```diff
@@ -18,7 +18,8 @@ plugins {
     checkstyle
     jacoco
     id("com.rameshkp.openapi-merger-gradle-plugin") version "1.0.4"
-    id ("org.eclipse.dataspaceconnector.dependency-rules") apply(false)
+    id("org.eclipse.dataspaceconnector.dependency-rules") apply (false)
+    id("org.eclipse.dataspaceconnector.test-summary") apply (false)
```

Comment:
I think this is not required for the custom plugins (for both dependency-rules and test-summary)